test/phone_form_field_test.dart

```diff
@@ -62,6 +64,20 @@ void main() {
         await tester.pumpWidget(getWidget());
         expect(find.byType(CircleFlag), findsWidgets);
       });
+
+      testWidgets('enabled, country chip should be enabled', (tester) async {
+        await tester.pumpWidget(getWidget(enabled: true));
+        final countryChip =
+            tester.widget<CountryCodeChip>(find.byType(CountryCodeChip));
+        expect(countryChip.enabled, true);
```

Comment:
We probably want to test that when disabled, tapping on the country code does not do anything. 
 
Currently those two test do not really cover much, so are, imo, unnecessary.